log/logger_test.go

```diff
@@ -367,26 +370,22 @@ func TestInfo(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	Info(sl, "Hello")
-	<-w.Finished
-	contents := w.Read()
-
-	if !strings.Contains(contents, "Hello") {
-		t.Errorf("received: '%v' but expected: '%v'", contents, "Hello")
-	}
+	Infof(nil, "%s", "hello")
 
 	Infof(sl, "%s", "hello")
 	<-w.Finished
-	contents = w.Read()
+	contents := w.Read()
 	if !strings.Contains(contents, "hello") {
 		t.Errorf("received: '%v' but expected: '%v'", contents, "hello")
 	}
 
+	Infoln(nil, "hello", "goodbye")
+
 	Infoln(sl, "hello", "goodbye")
 	<-w.Finished
 	contents = w.Read()
-	if !strings.Contains(contents, "hello goodbye") {
-		t.Errorf("received: '%v' but expected: '%v'", contents, "hello goodbye")
+	if !strings.Contains(contents, "hellogoodbye") {
+		t.Errorf("received: '%v' but expected: '%v'", contents, "hellogoodbye")
```

Comment:
This could be updated somewhat, you're sending multiple logs with the same msg and then testing if its there, not really distinguishing on whether anything was successful. I think you could do a bit more to ensure that the valid log is tested